login.css

```diff
@@ -1,7 +1,6 @@
 /* Import  font from google fonts*/
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
```

Comment:
20/08/2021: I trailing some font and the font I decided to use are Open sans and source sans pro. I import font from google fonts. The font for normal text is Open Sans and the fonts for heading will be source sans pro.